lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -454,6 +454,11 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
+            <?php if ($show_social_share_link) : ?>
```

Comment:
sorry I meant the other value I mentioned: `$campaign->variables['show_social_share_link']`
referring to here: https://github.com/DoSomething/phoenix/blob/c41162ae39e2e7c562fe9674b44bd44fcc00d0e5/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L445-L447